code/addons/controls/src/ControlsPanel.tsx

```diff
@@ -21,24 +16,26 @@ interface ControlsParameters {
   sort?: SortType;
   expanded?: boolean;
   presetColors?: PresetColor[];
+
+  /** @deprecated No longer used, will be removed in Storybook 8.0 */
   hideNoControlsWarning?: boolean;
 }
 
 export const ControlsPanel: FC = () => {
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
Minor suggestion here:

```suggestion
  const [isLoading, setIsLoading] = useState(!previewInitiated);
```

It requires that you move this below `useStorybookState()`.

Reply:
Thanks. I made the change.